index.js

```diff
@@ -1,5 +1,9 @@
 import configureEnv from "./config/configureEnv.js";
 import * as dotenv from "dotenv";
 dotenv.config();
+//test new branch
+// console.log(process.env);
 
-configureEnv();
+configureEnv().then(() => {
+  console.log(process.env);
+});
```

Comment:
wrap this in another function and run it immediately. this way you can use `await` and also try/catch the whole thing. this is a common technique when you need to execute asynchronous code in a top-level scope (happens a lot in index.js, when youre doing stuff like configuring the app (also think of spinning up a db client early)

```js
(async () => {
  try {
    await configureEnv();
  } catch(err) {
    // ...
  }
})();
```